src/sentry/conf/server.py

```diff
@@ -78,6 +78,11 @@ def env(key, default="", type=None):
 # List of IP subnets which should not be accessible
 SENTRY_DISALLOWED_IPS = ()
 
+# When resolving DNS for external sources (source map fetching, webhooks, etc),
+# ensure that domains are fully resolved first to avoid poking internal
+# search domains.
+SENTRY_ENSURE_FQDN = False
```

Comment:
Defaulting to False for the same reason we default to False for `SENTRY_ALLOWED_IPS` since there's a good change that on-premise would not want this behavior.